client/src/components/JurisdictionAdmin/BatchRoundSteps/EnterTalliesStep.tsx

```diff
@@ -12,9 +12,12 @@ import { Column, Row } from '../../Atoms/Layout'
 
 const Panel = styled(StepPanel)`
   padding: 0;
+  border-top: 1px solid ${Colors.LIGHT_GRAY2}; /* Match ListAndDetail */
+  border-bottom: 1px solid ${Colors.LIGHT_GRAY2};
```

Comment:
Thanks for getting to this! I know I said I would 😅, just had it low on my list of priorities